docusaurus.config.js

```diff
@@ -231,15 +231,8 @@ const config = {
                 html: `<div class="navbar__client__dropdown"><div class="navbar__client__dropdown__icon"><img src="/img/shield.svg" alt="Shield icon" /></div>
                 <div class="navbar__client__dropdown_text"><div class="text-base text-semibold">Code of conduct</div>`,
               },
-
             ],
           },
-          {
```

Comment:
If we remove this IMO we need to replace it with another CTA, as this was the lowest friction way for a website visitor to have an interactive experience.